samtranslator/schema/aws_serverless_httpapi.py

```diff
@@ -141,6 +141,9 @@ class Globals(BaseModel):
 class Resource(BaseModel):
     Type: Literal["AWS::Serverless::HttpApi"]
     Properties: Optional[Properties]
-    Metadata: Optional[PassThroughProp]
     Condition: Optional[PassThroughProp]
     Connectors: Optional[Dict[str, EmbeddedConnector]]
+    UpdateReplacePolicy: Optional[PassThroughProp]
```

Comment:
nit: if all of them have the same resource attributes, we could have a common class that online defines the resource attributes, and then `Resource`s inherit from it; less typing/risk